### ISSUE TITLE
Naming Service cache refreshing

### ISSUE THREAD

**Issue by kubawn** [Microsoft/service-fabric#69]
All of the overloads of the ServicePartitionResolver.ResolveAsync call the same overload of the FabricClient.ServiceManagementClient.ResolveServicePartitionAsync, while at least two of those that take ResolvedServicePartition should call one of the overloads that refresh the cache.

**Comment by suchiagicha** [comment]
@kubawn 
Yes , they call same overload. So ServicePartitionResolver API which doesn't take Previous RSP , pass null to the FabricClient.ServiceManagementClient.ResolveServicePartitionAsync, Api . For other ServicePartitionResolver Api which takes previous RSP, we pass same to fabric Client.
If you pass previous RSP to  ServicePartitionResolver  API, cache will be refreshed
